packages/create-plugin/src/utils/utils.prettifyFiles.ts

```diff
@@ -1,16 +1,19 @@
 import { exec as nodeExec } from 'node:child_process';
 import { promisify } from 'node:util';
 import fs from 'node:fs';
+import { getPackageJson } from './utils.packagejson.js';
 
 const exec = promisify(nodeExec);
 
-export async function prettifyFiles(exportPath: string) {
+export async function prettifyFiles(exportPath: string, projectRoot = exportPath) {
```

Comment:
Where is `projectRoot` used within this function?

Reply:
You were right, the last version was wrong and I forgot to use the `projectRoot`.

This was fixed and `projectRoot` is used in line 13

```ts
 const prettierVersion = getPrettierVersion(projectRoot);
```

This is necessary because `prettifyFiles` is called in two different ways, in plugin generate, when `exportPath` is the same as `projectRoot` and in plugin update when exportPath is `.config` and `projectRoot` is the current command location.